javascript/modules/client/src/internal/encoding/plugins.ts

```diff
@@ -19,6 +19,17 @@ import {
   IPluginSettings,
 } from "../interfaces/plugins";
 
+import { Interface } from "ethers/lib/utils";
```

Comment:
Could we make sure that we're using the scoped package? `@ethersproject/...`  instead of `ethers`

Otherwise, we should have installed the full `ethers` NPM package, which is quite large.
The same applies everywhere else